storage/mysql/map_storage.go

```diff
@@ -124,21 +124,22 @@ func (m *mapTX) Set(keyHash []byte, value trillian.MapLeaf) error {
 	}
 	defer stmt.Close()
 
-	_, err = stmt.Exec(m.ms.mapID, []byte(keyHash), m.writeRevision, flatValue)
+	_, err = stmt.Exec(m.ms.mapID, keyHash, m.writeRevision, flatValue)
 	return err
 }
 
-func (m *mapTX) Get(revision int64, keyHashes [][]byte) ([]trillian.MapLeaf, error) {
-	stmt, err := m.ms.getStmt(selectMapLeafSQL, len(keyHashes), "?", "?")
+// TODO: API problem: there's nothing to say which map leaves correspond to which indexes.
```

Comment:
`MapLeaf.Index` tells (or should) that.
As we spoke about earlier there's still some fiddling to do with the API though, either removing `IndexValue` or replacing `[]MapLeaf` with `map[_something_]=MapLeaf`... Maybe update the `TODO` slightly?